src/ansiblelint/utils.py

```diff
@@ -201,6 +202,13 @@ def ansible_template(
     re_valid_filter = re.compile(r"^\w+(\.\w+\.\w+)?$")
     templar = ansible_templar(basedir=basedir, templatevars=templatevars)
 
+    # Skip lookups for ansible-core >= 2.19; use disable_lookups for older versions
+    if "lookup" in varname:
```

Comment:
You should also check for `query` and `q`: https://docs.ansible.com/ansible/latest/plugins/lookup.html#forcing-lookups-to-return-lists-query-and-wantlist-true

Reply:
I created a new issue for this: #4695.